### PR TITLE
DEVDOCS-2282: Fix Linux Install Instructions

### PR DESCRIPTION
**Pull request by aglensmith**
# [DEVDOCS-2282](https://jira.bigcommerce.com/browse/DEVDOCS-2282)

## What changed?
* Removed install npm and nodejs step since nvm will install both when you run `nvm install`. 
* Added links to stencil cli and nvm readme with instructions on getting latest info for both.

### CODE REVIEW

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 1]
```diff
@@ -126,19 +124,18 @@ npm install -g @bigcommerce/stencil-cli
...
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.0/install.sh | bash
+# Download and install nvm if you don't have it.
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.36.0/install.sh | bash
 
 # reload .bashrc so nvm command works
 source ~/.bashrc
 
-# Explicitly install supported node version
-nvm install 10.17
+# explicitly install and use supported node version
```
Comment (uwikaiddi) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:133]: Correct capitalization

**Review by uwikaiddi — CHANGES_REQUESTED** [review]

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md (hunk 1 above)
Comment (aglensmith) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:133]:
```suggestion
# Explicitly install and use supported node version
```

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 2]
```diff
@@ -126,19 +124,18 @@ npm install -g @bigcommerce/stencil-cli
...
 
-# Explicitly install supported node version
-nvm install 10.17
+# Explicitly install and use supported node version
+nvm install 12
 
-# Install Stencil CLI
+nvm use 12
+
+# install stencil
```
Comment (aglensmith) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:138]:
```suggestion
# Install stencil
```

File: docs/stencil-docs/installing-stencil-cli/installing-stencil.md [hunk 3]
```diff
@@ -126,19 +124,18 @@ npm install -g @bigcommerce/stencil-cli
...
 ```shell
-## Update package list, then install node and npm
-sudo apt-get update && sudo apt-get install nodejs npm
-
-# Download nvm install.sh and run with bash
-curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.31.0/install.sh | bash
+# Download and install nvm if you don't have it.
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.36.0/install.sh | bash
 
 # reload .bashrc so nvm command works
```
Comment (aglensmith) [inline docs/stencil-docs/installing-stencil-cli/installing-stencil.md:130]:
```suggestion
# Reload .bashrc so nvm command works
```

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**